GravatarApp/AvatarPicker/Avatar/AvatarActionsMenu.swift

```diff
@@ -11,23 +12,29 @@ struct AvatarActionsMenu<Label>: View where Label: View {
 
     func actionsMenu(isSelected: Bool, label: () -> Label) -> some View {
         Menu {
-            Section {
-                if !isSelected {
-                    button(for: .select)
-                }
-                button(for: .share)
-                // TODO: We might use this soon, so keeping it commented for now
-                /**
-                  if #available(iOS 18.2, *) {
-                      if EnvironmentValues().supportsImagePlayground {
-                          button(for: .playground)
+            Group {
+                Section {
+                    if !isSelected {
+                        button(for: .select)
+                    }
+                    button(for: .share)
+                    // TODO: We might use this soon, so keeping it commented for now
+                    /**
+                      if #available(iOS 18.2, *) {
+                          if EnvironmentValues().supportsImagePlayground {
+                              button(for: .playground)
+                          }
                       }
-                  }
-                  button(for: .altText)
-                 */
+                      button(for: .altText)
+                     */
+                }
+
+                Section {
+                    button(for: .delete)
+                }
             }
-            Section {
-                button(for: .delete)
+            .onAppear {
+                labelTapAction?()
```

Comment:
As an option, we can get the analytics instance from environment here and track the action menu tap right here 🤔 

I'm not sure if it's better, but at least it will avoid the chain of callbacks up unitl `AvatarPickerView`

Reply:
Yeah it's really a close call for this one. Just that, it felt more consistent to do this way with the existing actions. Also keeping it decoupled from business logic at this child view level felt slightly more right.